src/components/Sum.jsx

```diff
@@ -1 +1,14 @@
-// export function Sum() {}
+import React from 'react';
+
+export function Sum({ a, b }) {
+  return (
+    <>
+      <p>{`Sum of ${a} and ${b} is ${a + b}`}</p>
+    </>
```

Comment:
```suggestion
    <p>{`Sum of ${a} and ${b} is ${a + b}`}</p>
```
react fragment is redundant if the JSX you return has 1 parent (high-level) tag - FYI